### PR TITLE
Drafts sorting algorithm objectives

### PR DESCRIPTION
**Pull request by chrisbay**

### CODE REVIEW

**Comment by chrisbay** [comment]
[DS Lesson 3 Learning Objectives](https://trello.com/c/H9A9Lg9x/15-ds-lesson-3-learning-objectives)

File: data-structures-and-algorithms/lesson03.rst [hunk 1]
```diff
@@ -4,6 +4,15 @@ Data Structures and Algorithm Analysis - Lesson 3 Learning Objectives
 Goals
 -----
 
+- Understand a few common sorting algorithms
+- Compute and understand the big-O efficiency of common sorting algorithms, along with their best and worst-case scenarios
```
Comment (the-vampiire) [inline data-structures-and-algorithms/lesson03.rst:8]: are we also covering best case? from our meeting i was under the impression that we only cover worst case in this course.

is best case only for this sorting algos chapter? or should it be introduced in the earlier chapters as well?

**Review by the-vampiire — COMMENTED** [review]
notes for consideration

File: data-structures-and-algorithms/lesson03.rst (hunk 1 above)
Comment (chrisbaylewis) [reply data-structures-and-algorithms/lesson03.rst:8]: We should discuss best-case, but for sorting algorithms only. We don't need to express this in little-o notation or anything, but just get students thinking about what happens for a given algorithm if the set is already sorted, for example.

**Pull request merged**